engine/sync_tx_service.go

```diff
@@ -253,18 +242,14 @@ func processSyncTxSave(ctx context.Context, txInfo *chainstate.TransactionInfo,
 	}
 
 	transaction.setChainInfo(txInfo)
-
-	message := "transaction was found on-chain by " + chainstate.ProviderBroadcastClient
-
 	if err := transaction.Save(ctx); err != nil {
-		_bailAndSaveSyncTransaction(
-			ctx, syncTx, SyncStatusError, syncActionSync, "internal", err.Error(),
-		)
+		_addSyncResult(ctx, syncTx, syncActionSync, "internal", err.Error())
 		return err
 	}
 
+	message := "transaction was found on-chain by " + chainstate.ProviderBroadcastClient
```

Comment:
Because of the message local variable is used only once, I believe we could put it linline, like this:
```golang
StatusMessage: "transaction was found on-chain by " + chainstate.ProviderBroadcastClient,
```

Reply:
changed